### ISSUE TITLE
Error launching GUI - no module named atlaselectrophysiology

### ISSUE THREAD

**Issue by cspurandare** [int-brain-lab/iblapps#114]
Hello,
I am trying to test out the brain region registration GUI, and working through Anaconda on a Windows 11 PC. After following installation instructions for the external user, I am getting the error of no module named atlaselectrophysiology, but that folder is present and is actually the folder where the ephys_atlas_gui.py file resides. Is this a known issue or am I missing some step?
The screenshots for the command window with the error, and my folder for installation are attached, 
Do let me know,
Thanks in advance,
Chinmay
![Screenshot 2024-02-23 223930](https://github.com/int-brain-lab/iblapps/assets/39069923/aaa23740-5e1a-4b0f-a795-060c12069d1c)
![Screenshot 2024-02-23 223915](https://github.com/int-brain-lab/iblapps/assets/39069923/a7d76aab-3c76-48df-a80d-9ef348fe223f)

**Comment by mayofaulkner** [comment]
Hello! Could you copy and paste the output of running `pip list`

**Comment by cspurandare** [comment]
[output_Feb26_2024.txt](https://github.com/int-brain-lab/iblapps/files/14410321/output_Feb26_2024.txt)
Hi Mayo,
Thank you for your prompt reply. Attached is the output for pip list in base. Also as screenshots is attached output of pip list in different folders. I think I can guess what the problem is - the module is not in the list of identified modules by pip? 
Thanks and do let me know how to proceed,
Chinmay
![Screenshot 2024-02-26 115802](https://github.com/int-brain-lab/iblapps/assets/39069923/e1eba342-e1c8-4fb5-bd55-7bec9977b238)

**Comment by mayofaulkner** [comment]
Okay, it looks it hasn't installed properly.
Can you run this step in the installation instructions again 
```
pip install --editable iblapps
```

If you get any errors when installing can you print the output here so I can take a look.

Thanks

**Comment by cspurandare** [comment]
Hi Mayo,
Thank you so much for your help. I think that was the issue, I redid installation and this time I also did some of the installation steps from the first part about IBL user instructions -
from the link https://github.com/int-brain-lab/iblenv to install a unified environment that is compatible with all IBL libraries being developed. Now the GUI works and I checked that later steps upto (Upload alignment) works successfully.
Thank you again and I am looking forward to using the GUI, it seems very useful and intuitive !
Chinmay
